templates/travis/.travis/install.sh.j2

```diff
@@ -130,6 +135,45 @@ spec:
 {%- endif %}
 CRYAML
 
+if [ "$TEST" = 's3' ]; then
+  cat > deploy/crds/pulpproject_v1alpha1_pulp_cr.yaml << CRYAML
+  apiVersion: pulpproject.org/v1alpha1
+  kind: Pulp
+  metadata:
+    name: example-pulp
+  spec:
+    pulp_file_storage:
+      # k3s local-path requires this
+      access_mode: "ReadWriteOnce"
+      # We have a little over 40GB free on Travis VMs/instances
+      size: "40Gi"
+    image: {% if plugin_name != 'pulpcore' %}{{ plugin_name }}{% else %}pulp_file{% endif %}
+    tag: "${TAG}"
+    database_connection:
+      username: pulp
+      password: pulp
+      admin_password: pulp
+  {%- if pulp_settings or test_s3 %}
+    pulp_settings:
+      {% if pulp_settings %}
+      {%- for setting, value in pulp_settings.items() -%} {{ setting }}: {{ value }}
+      {% endfor -%}
+      {% endif -%}
+      aws_access_key_id: "AKIAIT2Z5TDYPX3ARJBA"
```

Comment:
I think these need to be wrapped in a `{%- if test_s3 %}` no?

Reply:
it is not needed as it is inside of `if [ "$TEST" = 's3' ]; then`

Reply:
Yup, I am blind.